reana_job_controller/job_monitor.py

```diff
@@ -241,6 +261,85 @@ def watch_jobs(self, job_db, app):
                 time.sleep(120)
 
 
+slurmJobStatus = {
+    'failed': ['BOOT_FAIL', 'CANCELLED', 'DEADLINE', 'FAILED', 'NODE_FAIL',
+               'OUT_OF_MEMORY', 'PREEMPTED', 'TIMEOUT', 'SUSPENDED',
+               'STOPPED'],
+    'succeeded': ['COMPLETED'],
+    'running': ['CONFIGURING', 'COMPLETING', 'RUNNING', 'STAGE_OUT'],
+    'idle': ['PENDING', 'REQUEUE_FED', 'REQUEUE_HOLD', 'RESV_DEL_HOLD',
+             'REQUEUED', 'RESIZING']
+    # 'REVOKED',
+    # 'SIGNALING',
+    # 'SPECIAL_EXIT',
+}
+
+
+@singleton
+class JobMonitorSlurmCERN(JobMonitor):
+    """Slurm jobs monitor CERN."""
+
+    def __init__(self, app=None):
+        """Initialize Slurm job monitor thread."""
+        super(__class__, self).__init__(
+            thread_name='slurm_job_monitor'
+        )
+
+    def format_slurm_job_query(self, backend_job_ids):
+        """Format Slurm job query."""
+        cmd = 'sacct --jobs {} --noheader --allocations --parsable ' \
```

Comment:
Spotted after latest repush:

```
2019-12-03 17:02:07,872 | root | slurm_job_monitor | ERROR | No slurm jobs
2019-12-03 17:02:07,923 | root | slurm_job_monitor | ERROR | Exception while executing cmd: sacct --jobs  --noheader --allocations --parsable --format State,JobID 
sacct: fatal: Bad job/step specified: --noheader
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/reana_job_controller/utils.py", line 90, in exec_command
    raise Exception(stderr.read().decode("utf-8"))
Exception: sacct: fatal: Bad job/step specified: --noheader

2019-12-03 17:02:07,924 | root | slurm_job_monitor | ERROR | Unexpected error: 'NoneType' object has no attribute 'rstrip'
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/reana_job_controller/job_monitor.py", line 319, in watch_jobs
    for item in stdout.rstrip().split('\n'):
AttributeError: 'NoneType' object has no attribute 'rstrip'
2019-12-03 17:02:09,932 | werkzeug | Thread-157 | INFO | 127.0.0.1 - - [03/Dec/2019 17:02:09] "GET /jobs/5f481d60-efcf-439a-ad9f-fc1684273d50 HTTP/1.1" 2
```

The job continues and finishes successfully...

Reply:
Fixed :)